refactor(transformer/using): re-order imports

overlookmotel opened the pull request:
Follow-on after #9310. Style nit. Re-order imports with external crates before `oxc_*` crates.

This is just my personal OCD preference! I find it easier to see where imports are coming from when ordered like this. Sadly there is no rustfmt rule to enforce this style.

overlookmotel commented:
* **#9677** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9677?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#9676** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9676?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#9675** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9675?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#9674** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9674?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#9673** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9673?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#9672** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9672?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/9672?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=9672).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

camc314 commented:
### Merge activity

* **Mar 11, 5:37 AM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Mar 11, 5:37 AM EDT**: A user added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc).

overlookmotel committed:
refactor(transformer/using): re-order imports (#9672)

Follow-on after #9310. Style nit. Re-order imports with external crates before `oxc_*` crates.

This is just my personal OCD preference! I find it easier to see where imports are coming from when ordered like this. Sadly there is no rustfmt rule to enforce this style.
